lib/std/private/browserutils.nim

```diff
@@ -0,0 +1,87 @@
+##[
+Experimental API, subject to change
+]##
+
+#[
+TODO:
+
+## scratch
+# let dir = getTempDir() / "D20210227T155249" # PRTEMP mkstmp ?
+# fileHtml = getHtmlFile(nimcacheDir.querySetting, fileJs)
+]#
+
+from std/browsers import openDefaultBrowser
+import std/os
+import std/strformat
+from std/strutils import `%`
+import std/compilesettings
+
+# proc serveJsBrowser*(fileJs: string, port: int) =
+proc serveJsBrowser*(fileJs: string) =
+  let port = 7031 # PRTEMP
+  let jsname = "input.js"
+  let html = fmt"""
+<!DOCTYPE html>
+<html>
```

Comment:
```suggestion
<html>
<head>
<title>Nim<title>
</head>
```

Must have non empty title to be HTML valid.

Reply:
done